tests/migration_test.go

```diff
@@ -971,7 +971,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				tests.UpdateClusterConfigValueAndWait("migrations", originalMigrationConfig)
 			})
 			It("[test_id:2303][posneg:negative] should secure migrations with TLS", func() {
-				tests.SkipIpv6TestsFailingAfterVmImageUpgradeToFc32()
+				tests.SkipMigrationTestIfRunnigOnKindInfra()
```

Comment:
i am not sure my PR fix those 2, we need to verify
was SkipMigrationTestIfRunnigOnKindInfra were on master before SkipIpv6TestsFailingAfterVmImageUpgradeToFc32 was added ?

in case it was, its not right imo to replace skips, but need to cascade them,
i dont see i removed them from my PR #3992

Reply:
https://github.com/kubevirt/kubevirt/blob/fde14127e7e5b3035167a1a7a726c9a387fa7fa8/tests/migration_test.go#L964

see please that there wasnt a line there

Reply:
right this why added the Edit above,
not all migration, just the ones that i didnt add a new skip for them, 
due to stressTest or to NFS 
https://github.com/kubevirt/kubevirt/pull/3392/files#diff-a5a178b4c66835110ceff63c61d28c96R4664

see the two skipper here please, i have added an issue for them

Reply:
You don't need to add the migration skip, since this test is not failing on the issues the migration PR fixes (it is a negative test).

Reply:
> You don't need to add the migration skip, since this test is not failing on the issues the migration PR fixes (it is a negative test).

I see this line: `if strings.Contains(err.Error(), "remote error: tls: bad certificate")`
Doesn't that mean it expects to fail on a specific tls issue? For that a connection needs to be established (or attempted) and failing with that error.
Or do I understand this whole story wrongly?

Reply:
I don't know the story. But currently the lane is green without that test being skipped. Maybe the test has a bug and doesn't actually check what it has to check. But adding a skip and then removing it in the migration PR won't fix it.
Let's open a separate discussion to understand this test.

Reply:
I do not like it but I will comply if the test passes with this change.
Checking locally and will update accordingly.

Reply:
It passed locally on the ipv6 lane.